INFO: localdir at /localscratch/13628813.michal.kubacki
INFO: your job will run on local system.
root_dir: /group/testa/michal.kubacki/herring
out_dir: /group/testa/michal.kubacki/herring/output_hg19_all_cells
in_dir: /group/testa/michal.kubacki/herring/data
tmp_dir: /group/testa/michal.kubacki/herring/tmp
2024-07-17 18:23:25,281 cisTopic     INFO     Normalizing RNA data
WARNING: adata.X seems to be already log-transformed.
/home/michal.kubacki/.conda/envs/scenicplus/lib/python3.11/site-packages/scanpy/preprocessing/_scale.py:299: UserWarning: Received a view of an AnnData. Making a copy.
  view_to_actual(adata)
/home/michal.kubacki/.conda/envs/scenicplus/lib/python3.11/site-packages/scipy/sparse/_index.py:145: SparseEfficiencyWarning: Changing the sparsity structure of a csr_matrix is expensive. lil_matrix is more efficient.
  self._set_arrayXarray(i, j, x)
2024-07-17 18:28:12,865 cisTopic     INFO     Processing 1 query sample(s) using 1 cpu(s)
2024-07-17 18:28:30,927	INFO worker.py:1762 -- Started a local Ray instance. View the dashboard at [1m[32mhttp://127.0.0.1:8265 [39m[22m
[36m(label_transfer_ray pid=2703262)[0m /home/michal.kubacki/.conda/envs/scenicplus/lib/python3.11/site-packages/scanpy/preprocessing/_scale.py:299: UserWarning: Received a view of an AnnData. Making a copy.
[36m(label_transfer_ray pid=2703262)[0m   view_to_actual(adata)
[36m(label_transfer_ray pid=2703262)[0m /home/michal.kubacki/.conda/envs/scenicplus/lib/python3.11/site-packages/pycisTopic/label_transfer.py:241: FutureWarning: Use anndata.concat instead of AnnData.concatenate, AnnData.concatenate is deprecated and will be removed in the future. See the tutorial for concat at: https://anndata.readthedocs.io/en/latest/concatenation.html
[36m(label_transfer_ray pid=2703262)[0m   adata_concat = ref_anndata.concatenate(
[36m(label_transfer_ray pid=2703262)[0m 2024-07-17 18:28:51,814 cisTopic     INFO     Normalizing ATAC data from sample multiome_brain
[36m(label_transfer_ray pid=2703262)[0m 2024-07-17 18:29:49,186 cisTopic     INFO     Running integration with scanorama
[36m(label_transfer_ray pid=2703262)[0m Found 595 genes among all datasets
[36m(label_transfer_ray pid=2703262)[0m [[0.         0.14091326]
[36m(label_transfer_ray pid=2703262)[0m  [0.         0.        ]]
[36m(label_transfer_ray pid=2703262)[0m Processing datasets (0, 1)
Traceback (most recent call last):
  File "/home/michal.kubacki/Githubs/Re-MEND/code/External_Datasets/GeneSet_Derivation/Herring_scenic/PycisTopic_Scenic/label_transfer.py", line 112, in <module>
    sns.heatmap(conf_mat.loc[conf_mat.columns], ax = ax)
                ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/home/michal.kubacki/.conda/envs/scenicplus/lib/python3.11/site-packages/pandas/core/indexing.py", line 1074, in __getitem__
    return self._getitem_axis(maybe_callable, axis=axis)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/michal.kubacki/.conda/envs/scenicplus/lib/python3.11/site-packages/pandas/core/indexing.py", line 1302, in _getitem_axis
    return self._getitem_iterable(key, axis=axis)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/michal.kubacki/.conda/envs/scenicplus/lib/python3.11/site-packages/pandas/core/indexing.py", line 1240, in _getitem_iterable
    keyarr, indexer = self._get_listlike_indexer(key, axis)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/michal.kubacki/.conda/envs/scenicplus/lib/python3.11/site-packages/pandas/core/indexing.py", line 1433, in _get_listlike_indexer
    keyarr, indexer = ax._get_indexer_strict(key, axis_name)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/michal.kubacki/.conda/envs/scenicplus/lib/python3.11/site-packages/pandas/core/indexes/base.py", line 6108, in _get_indexer_strict
    self._raise_if_missing(keyarr, indexer, axis_name)
  File "/home/michal.kubacki/.conda/envs/scenicplus/lib/python3.11/site-packages/pandas/core/indexes/base.py", line 6171, in _raise_if_missing
    raise KeyError(f"{not_found} not in index")
KeyError: "['CGE_dev', 'ID2', 'LAMP5_NOS1', 'PV_SCUBE3', 'Vas'] not in index"
slurmstepd: error: _cgroup_procs_check: failed on path /sys/fs/cgroup/memory/slurm/uid_10666/job_13628813/step_batch/cgroup.procs: No such file or directory
slurmstepd: error: unable to read '/sys/fs/cgroup/memory/slurm/uid_10666/job_13628813/step_batch/cgroup.procs'
